clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -256,7 +261,7 @@ export const ConnectorParametersForm = ({
 
     if (_.isEmpty(initialValues.enabled_actions)) {
       initialValues.enabled_actions = connectionOption.supported_actions.map(
-        (action) => action.toString(),
+        (action) => action.toString()
```

Comment:
woops, it seems my frontend setup is not quite working ok. TODO:  undo these changs